mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -311,7 +316,6 @@ func resourceClusterUpdate(d *schema.ResourceData, meta interface{}) error {
 	if requestUpdate {
 		// Set read-only fields to an empty string to make the API happy
 		c.StateName = ""
-		c.MongoDBVersion = ""
```

Comment:
hey @supersol any reason why you removed this line? I had trouble with the atlas API back in the day when I tried to change the `MongoDBVersion`

Reply:
Hey @akshaykarle. Good point, I could mixed up it with mongo major version.